cmake/onnxruntime_unittests.cmake

```diff
@@ -162,6 +162,11 @@ file(GLOB onnxruntime_test_providers_src CONFIGURE_DEPENDS
 file(GLOB_RECURSE onnxruntime_test_providers_cpu_src CONFIGURE_DEPENDS
   "${TEST_SRC_DIR}/providers/cpu/*"
   )
+
+if(onnxruntime_DISABLE_ML_OPS)
+  list(FILTER onnxruntime_test_providers_cpu_src EXCLUDE REGEX ".*/ml/.*")
```

Comment:
shouldn't you also do something similar in onnxruntime_providers.cmake to exclude ml sources from being built?

Reply:
Sure, but strictly from binary size reduction point of view it doesn't matter since LTO removes all unused code from the binary. I'll fix it.